core/domain/suggestion_registry.py

```diff
@@ -466,6 +466,35 @@ def accept(self, commit_message):
             self.final_reviewer_id, self.target_id, [self.change],
             commit_message, is_suggestion=True)
 
+    def get_all_html_content_strings(self):
```

Comment:
No, we are not checking or migrating suggestions of type SuggestionEditStateContent.
Because these suggestions don't currently have RTEs, also in the future we don't plan to add them.
Hence we don't need this there.

Reply:
done , added the method

Reply:
Actually, the only suggestion with production data is probably SuggestionEditStateContent. The other suggestions are part of the community dashboard which was released, but I am not sure if they have data yet... This is just FYI...